cmd/influx/cli/cli.go

```diff
@@ -663,23 +663,24 @@ func (c *CommandLine) Settings() {
 
 func (c *CommandLine) help() {
 	fmt.Println(`Usage:
-        connect <host:port>   connect to another node
-        auth                  prompt for username and password
-        pretty                toggle pretty print
-        use <db_name>         set current databases
-        format <format>       set the output format: json, csv, or column
-        precision <format>    set the timestamp format: h,m,s,ms,u,ns
-        consistency <level>   set write consistency level: any, one, quorum, or all
-        settings              output the current settings for the shell
-        exit                  quit the influx shell
-
-        show databases        show database names
-        show series           show series information
-        show measurements     show measurement information
-        show tag keys         show tag key information
-        show tag values       show tag value information
-
-        a full list of influxql commands can be found at:
+        connect <host:port>   Connect connects to another node specified by host:port
```

Comment:
I'm not sure I like this format of continually repeating the command in the explanation -- it seems overly verbose. I preferred the older format.

@beckettsean